prompts/opencode/burlaki-human/SKILL.md

```
@@ -1,30 +1,101 @@
+1) Parse args from user input. Support `--resolve <story-id>`, optional `--note "text"`, `--list-tags`, and optional `--tag <tag>` (default `@human`). If `--resolve` is provided, run the resolve flow. Else if `--list-tags` is provided, run the list-tags flow. Otherwise run the view escalations flow.
 
-1) Search in PRD/task artifacts (only include paths that exist):
+2) **Resolve flow** (`/burlaki-human --resolve <story-id> [--note "text"]`):
+   1. Validate PRD exists at `.agents/tasks/prd.json`. If missing: exit 2 with `No PRD found. Run /burlaki-start first.`
+   2. Parse PRD, find story by id. If parse error: exit 1 and show error details.
+   3. If not found: exit 2 with `Story not found: <id>`.
+   4. If story.status != `blocked`: exit 2 with `Story <id> is not blocked (status: <status>)`.
+   5. Create checkpoint (allow empty if tree clean):
+       `git add -A && git commit --allow-empty -m "[burlaki-checkpoint] Resolved escalation: <story-id>"`
+   6. Atomic write PRD (update story fields):
+       ```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)